src/components/card/HomeCard.tsx

```diff
@@ -8,7 +8,11 @@ import axios from "axios";
 function HomeCard(props: any) {
   console.log(props);
   const backStyle = {
-    backgroundImage: `url(${props.owner_avatar})`,
+    backgroundImage: `url(${
```

Comment:
Updated the `HomeCard` to actually display the right info along with the users avatar or if none a default image